sequencer/src/api/fs.rs

```diff
@@ -33,9 +33,9 @@ impl SequencerDataSource for DataSource {
         let mut index = Index::default();
 
         // Index blocks by timestamp.
-        let mut blocks = data_source.get_block_range(..).await?;
```

Comment:
Semantics of `get_block_range` have changed with an unbounded range. It will now continue to fetch blocks as long as you keep polling the stream, which is not what we want here. Instead you'll want something like `data_source.get_block_range(..data_source.block_height().await?).await?`.